guides/common/modules/con_active-directory-with-cross-forest-trust.adoc

```diff
@@ -4,8 +4,8 @@
 Kerberos can create `cross-forest trust` that defines a relationship between two otherwise separate domain forests.
 A domain forest is a hierarchical structure of domains; both AD and {FreeIPA} constitute a forest.
 With a trust relationship enabled between AD and {FreeIPA}, users of AD can access Linux hosts and services using a single set of credentials.
-ifndef::orcharhino[]
-For more information on cross-forest trusts, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/windows_integration_guide/active-directory-trust[Creating Cross-forest Trusts with Active Directory and Identity Management] in the _{RHEL} Windows Integration_ guide.
+ifdef::satellite[]
+For more information on cross-forest trusts, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/planning_identity_management/planning-a-cross-forest-trust-between-idm-and-ad_planning-identity-management[Planning a cross-forest trust between IdM and AD] in _{RHEL} Planning Identity Management_.
```

Comment:
```suggestion
For more information on cross-forest trusts, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/planning_identity_management/planning-a-cross-forest-trust-between-idm-and-ad_planning-identity-management[Planning a cross-forest trust between IdM and AD] in _Planning Identity Management_.
```
I know this has been the norm for suggesting that the guide is not from satellite documentation, but I propose to stop following that norm. When referring to Satellite guides, we don't say "see ... in the **Satellite** _Managing Hosts_ guide". I am not forcing it of course, so consider it optional, but I would love to hear your opinion on this.

Reply:
@adamlazik1 I think it can serve the purpose of informing the user to search the RHEL docs, for IdM related information. Since, we are within the Satellite docs, it becomes redundant to use the product label in the case of any other Sat guide. I'm just thinking out loud here, maybe @Lennonka can weigh in?

Reply:
My reasoning is that user does not have to search for the guide manually but just clicks on the link provided, therefore user does not need to distinguish between satellite and non-satellite guides. Therefore, I think using the exact name of the guide serves better readability. Since you mentioned redundancy, many RHEL guides already contain RHEL in the title so adding {RHEL} does not make sense from the reader's point of view. As I said previously, I am not forcing this change, but I would be interested to know what other writers think about this.

Reply:
I'd prefer to include the product label before the guide name, because it tells the user that it takes them to another product. I agree with Akshay that in case of Satellite, that would be redundant, but for other products I think it's useful to signify that.